scripts/tests/test_validate_docstrings.py

```diff
@@ -596,6 +614,19 @@ def return_not_documented(self):
         """
         return "Hello world!"
 
+    def return_not_documented_some_bare_returns(self):
```

Comment:
I don't think this adds value as a test since it's just not something you'd ever seen in code, though I think the intention is good.

Can you add a conditional where the first returns one of the first 4 examples and the second branch returns the string?

Reply:
Done.